### ISSUE TITLE
Big file upload suggestion

### ISSUE THREAD

**Issue by PeopleInside** [osTicket/osTicket#3615]
### Description

Suggest to not ask to the user to keep the ticket locked when is in progress a big attachments upload. Ticket should stay locked if a big upload is in progress because maybe the operator are not in front on the PC during a big upload.

Also another issue is that log in can expire on long big file upload, if this happen the ticked can be saved and sent.

### Steps to Reproduce

1. Create a ticket
2. Upload a file bigger up to 200 MB

A message is showed and required the operator click on renew button for have the ticked locked but big file upload is in progress and operator can be out of the PC

also session login session can expire and ticket will be losed. Maybe log in should not expire if upload is in progress?

A message is showed to keep the ticket on hold

Browser Firefox 50
OSTicket version the last 1.10

**Comment by fdellwing** [comment]
Most admins will not even allow uploads that big. I can see the problem here, but dont think many people are facing this problem.

**Comment by PeopleInside** [comment]
The issue was once I set PHP to allow big files OsTicket let me select support of 2 GB attachments. So maybe if OsTicket has attachments file limit should be set this as maximum also if the server can support bigger upload. I think.

Also I tried to upload a .mp4 file, the file was accepted to the upload also if this file was not in extensions supported, maybe the warning is showed only after uploaded 2 GB. The alert should be visible before the upload start: file not supported.

Also if I added .mp4 to extension was not able to upload into OsTicket, I need use an external service to use big upload.
